### PR TITLE
FullCalendar Integration

### PR DESCRIPTION
**Pull request by ProLoser**
on behalf of @joshkurz

### CODE REVIEW

File: modules/directives/fullcalendar/fullcalendar.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
+*  Implimentation of JQuery FullCalendar inspired by http://arshaw.com/fullcalendar/
```
Comment (boneskull) [inline modules/directives/fullcalendar/fullcalendar.js:2]: Implementation

File: modules/directives/fullcalendar/fullcalendar.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                            theme: true,
+                            header: {
+                                left: 'prev,next today',
+                                center: 'title',
+                                right: 'month,agendaWeek,agendaDay'
+                            },
+        
+                            editable: true,
+                            slotMinutes: 15,
+
```
Comment (boneskull) [inline modules/directives/fullcalendar/fullcalendar.js:48]: Seems like the above options should be stored in ui.config as defaults?

File: modules/directives/fullcalendar/fullcalendar.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                            // add event name to title attribute on mouseover. Dont really need this, but its cool. 
+                            eventMouseover: function(event, jsEvent, view) {
+                                if (view.name !== 'agendaDay') {
```
Comment (boneskull) [inline modules/directives/fullcalendar/fullcalendar.js:51]: should use ===

File: modules/directives/fullcalendar/fullcalendar.js (hunk 2 above)
Comment (ProLoser) [inline modules/directives/fullcalendar/fullcalendar.js:48]: The entire implementation needs some cleanup. This PR is not at all ready to be merged. I'm just forwarding a PR that was opened against my fork. I asked him to clean up a lot of different things (including removing the build files) but I have yet to hear back from him.

File: modules/directives/fullcalendar/fullcalendar.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
```
Comment (boneskull) [inline modules/directives/fullcalendar/fullcalendar.js:0]: Both this file and the spec file should conform to our new and improved code standarsds; that is to say, 2 spaces per indent.  Thanks!

Comment (joshkurz) [inline modules/directives/fullcalendar/fullcalendar.js:0]: I will git on these changes.

Thanks
Josh Kurz

**Pull request closed without merging**